chrome/content/zotero/xpcom/itemTreeManager.js

```diff
@@ -197,6 +197,12 @@ class ItemTreeManager {
 			options.forEach(opt => this._addColumns(opt, true));
 			return true;
 		}
+		// Turn defaultIn, disabledIn, and zoteroPersist into Set
+		["defaultIn", "disabledIn", "zoteroPersist"].forEach(key => {
+			if (options[key]) {
+				options[key] = new Set(options[key]);
+			}
+		});
```

Comment:
This is the wrong place to do this. First, now our own definition of columns still uses Sets, so the reasonable "example" that a plugin developer would use doesn't match the documentation. Secondly, if we convert our own definition to use arrays it will break the code. We need to convert our own definition of columns to use Arrays, and change the code which uses these options to expect arrays.